hathor/builder/builder.py

```diff
@@ -158,16 +164,19 @@ def build(self) -> BuildArtifacts:
             consensus_algorithm=consensus_algorithm,
             peer_id=peer_id,
             tx_storage=tx_storage,
+            event_storage=event_storage,
             network=self._network,
             wallet=wallet,
             rng=self._rng,
             checkpoints=self._checkpoints,
             capabilities=self._capabilities,
             environment_info=get_environment_info(self._cmdline, peer_id.id),
+            event_manager=event_manager,
```

Comment:
Are we always instantiating an event manager? I thought it could be disabled.

Reply:
No, it can be disabled. The `_get_or_create_event_manager()` method in the `Builder` may return `None` if the `EventManager` was not set with `set_event_manager()` neither enabled with `enable_event_manager()`. The `HathorManager` also receives an `Optional[EventManager]`. Should it be passed in the kwargs instead?

Reply:
I guess it is fine. Thanks!